sub/osd.c

```diff
@@ -58,7 +58,7 @@ static const m_option_t style_opts[] = {
     {"border-size", OPT_ALIAS("outline-size"), .alias_use_prefix = true},
     {"shadow-offset", OPT_FLOAT(shadow_offset)},
     {"spacing", OPT_FLOAT(spacing), M_RANGE(-10, 10)},
-    {"margin-x", OPT_INT(margin_x), M_RANGE(0, 300)},
+    {"margin-x", OPT_INT(margin_x), M_RANGE(0, 1200)},
```

Comment:
There's no reason to have this upper limit at all, could you change this to `INT_MAX`? Same for `margin-y`.